src/bot/bot.py

```diff
@@ -35,13 +36,28 @@ def __init__(self) -> None:
             WSTETH_B,
             STECRV_A,
         )
+        self.prices = {
+            "wstETH_A": 0,
+            "wstETH_B": 0,
+            "steCRV_A": 0
+        }
```

Comment:
You have the keys in `MakerIlk.symbol`.  So please you the constants. And you can use `defaultdict` instead of plain `dict`.